loone_data_prep/forecast_scripts/Chla_merged.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+def loads_predicted(input_dir, output_dir):
+    """
+    Calculate Chlorophyll-a loads based on inflows and chlorophyll-a data.
+    
+    input_dir: Directory where the input files are located.
+    output_dir: Directory where the output files will be saved.
+    St_Yr, St_M, St_D: Start date (year, month, day).
+    En_Yr, En_M, En_D: End date (year, month, day).
+    """
+    
+    # Read forecast inflow file
+    Flow_df = pd.read_csv(f"{input_dir}/geoglows_flow_df_ens_01_predicted.csv")
```

Comment:
I'm guessing using ensemble 01 hard coded here is intentional, but I thought I would check

Reply:
Good catch. This should probably be an average or median of the ensembles. I'll add some TODO comments so we can merge it for now.

```suggestion
    # TODO: Should this be an average/median of all of the ensembles? worst case?
    Flow_df = pd.read_csv(f"{input_dir}/geoglows_flow_df_ens_01_predicted.csv")
```